### PR TITLE
Re-enable broken unit tests

### PR DESCRIPTION
**Pull request by chfast**
The "rescue" test is still marked with 1 expected failure.

### CODE REVIEW

**Comment by chfast** [comment]
@winsvega what do you think?

File: test/unittests/libethereum/BlockChain.cpp [hunk 1]
```diff
@@ -476,7 +442,7 @@ BOOST_AUTO_TEST_CASE(insertException)
 	}
 }
 
-BOOST_AUTO_TEST_CASE(rescue)
+BOOST_AUTO_TEST_CASE(rescue, *utf::expected_failures(1))
```
Comment (winsvega) [inline test/unittests/libethereum/BlockChain.cpp]: is this test constantly failing?

File: test/unittests/libethereum/BlockChain.cpp [hunk 2]
```diff
@@ -80,39 +79,6 @@ BOOST_AUTO_TEST_CASE(opendb)
 	BOOST_CHECK_EXCEPTION(BlockChain bc2(p, tempDirBlockchain.path(), WithExisting::Verify), DatabaseAlreadyOpen, is_critical);
 }
 
-//(rebuild)
```
Comment (chfast) [inline test/unittests/libethereum/BlockChain.cpp]: I don't know. It was dead for so long so I deleted it.

File: test/unittests/libethereum/BlockChain.cpp (hunk 1 above)
Comment (chfast) [inline test/unittests/libethereum/BlockChain.cpp]: No, but fails with some good probability. And it is not about synchronization (see removed sleep), I waited many minutes and it still can fail. Boost is fine with tests expected to fail actually not to fail.

**Pull request merged**